CHANGELOG.md

```diff
@@ -24,6 +24,7 @@ See [Releases](https://github.com/opencrvs/opencrvs-farajaland/releases) for rel
 - Deployment: Removes some deprecated deployment code around Elastalert config file formatting
 - Provisioning: Creates backup user on backup servers automatically
 - Provisioning: Update ansible Github action task version
+- Change condition of Number of previous births 
```

Comment:
@anamulhaquemollah could you move this under 1.5.0 section please?

Reply:
Will do it.